const.py

```diff
@@ -0,0 +1,8 @@
+from enum import Enum
+from enum import auto
+
+
+class Components(Enum):
+    PROJECT_MAIN_PAGE = 1
```

Comment:
any reason why this doesn't also use `auto()`?

Reply:
Not sure. Maybe I thought it would star at zero and didn't want that, but it starts at one.